### ISSUE TITLE
Identify method or property of undefined or mixed variable

### ISSUE THREAD

**Issue by rentalhost** [kalessil/phpinspectionsea#808]
Is hard to identify visually if your method call or property come from some real object or if it comes from an undefined or mixed variable. For instance:

```php
// Ok: method "test" not found (actually is a PS inspection).
$knownSource = new \stdClass;
$knownSource->test();

// Fail: method "test" do not receive and inspection error.
/** @var mixed $unknowSource */
$unknowSource = new \stdClass;
$unknowSource->test(); 
```

I think that `mixed` types should not be used as an object, so it should be an error too, then I think that it should be a valid inspection to you write a well connected source code.

**Comment by kalessil** [comment]
We are handling mixed in the same way: if it preset, almost all inspections are skipping analysis.
+ it's PhpDoc, implicit return types are not allowing it =)
